app/lib/api_authentication/by_access_token.rb

```diff
@@ -112,7 +113,7 @@ def enforce_access_token_permission
     end
 
     def verify_access_token_scopes
-      return true unless access_token
+      return true unless authenticated_token
```

Comment:
I think this should not be modified because there is nothing to verify when it is a provider_key